fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -107,6 +107,16 @@ export const LocationSuggest = forwardRef<
       },
     );
 
+    useEffect(() => {
+      /**
+       * Location suggestions vary depending on the hirer.
+       * Hence, we should clear the selected location when the hirer changes.
+       */
+      setSelectedLocation(undefined);
+      setPlaceholder('');
+      onClear?.();
+    }, [hirerId, onClear]);
```

Comment:
Does this reset the hirer ID whenever `onClear` changes? Is that desirable?

Reply:
I'm guessing you meant resets the location? Fixed this up so that it will only change if the `hirerId` changes

Reply:
🤦 Yes, thanks!